File: python/ubervan.py
from array import array
from car import Car
class UberVan(Car):
    typecaraccepted = []
    seatsMaterial = []

    def __init__(self, license, driver, typecaraccepted, seatsMaterial):
        super().__init__(license, driver)
        self.typecaraccepted = typecaraccepted
        self.seatsMaterial = seatsMaterial

File: php/uberVan.php
<?php
require_once('car.php');
class UberVan extends Car {
    public $brand;
    public $model;
    public $typeCarAccepted = array();
    public $seatsMaterial = array();


    public function __construct($license,$driver, $brand, $model) {
        parent::__construct($license, $driver);
        $this->brand = $brand;
        $this->model = $model;
    }
}

?>